### ISSUE TITLE
Should additional requirements be applied to trainees?

### ISSUE THREAD

**Issue by jiripetrzelka** [erasmus-without-paper/ewp-specs-api-factsheet#6]
With the extension of the omobilities API to traineeships, it is no longer clear whether the additional requirements defined in Factsheet apply to trainees as well. I can imagine that for example Portfolio could be required for traineeships but not for studies.

And if the mobility is short-term-blended or short-term-doctoral the list of requirements may differ as well.

If we are expected to parse Factsheet information by a script and ask students to upload specific documents, the details element is also of limited use.

What should we do if requirements differ for activity types / activity attributes as defined in https://github.com/erasmus-without-paper/ewp-specs-api-omobilities/commit/9ad1ee52a0aff15bb8bd06a0b7111dce25fc898e ?

Should we list only the minimal subset of requirements that apply to all activity types/attributes?

Or do you plan to extend this API and allow to define applicable activity types/attributes for each requirement?

**Comment by janinamincer-daszkiewicz** [comment]
We need an opinion of the business owners.

**Comment by lioliosn** [comment]
I believe that requirements should be connected to each activity type (studies or traineehips) and therefore allow for differentiation. Traineeships are usually more flexible, so they might not have so many structured requirements like studies.

**Comment by Belenchusky** [comment]
I agree with the proposal. It can be a good idea to separate the information related to each type of mobility because it is clear that they can be different.

**Comment by kamil-olszewski-uw** [comment]
Comment from IRO of University of Warsaw (supporting the proposal): "The studies cannot be compared with traineeships, because it is a completely different activity and the documents may also differ. Traineeships at UW are not maintained by IRO, but by the Career Office with their own procedures. But it can be different at each university."

**Comment by HU-Berlin-IRO** [comment]
HU Berlin: we agree as well that the specifics of mobility types have to be taken into account, not only studies vs. traineeships, but as well short-term etc. It may be advisable to identify what are common requirements as minimum standards per mobility type, and what possible additional elements/optional fields could be required. (eg for internships in specific study fields, eg Medicine, there might be additional elements)

**Comment by pleys** [comment]
This topic was discussed at a meeting with Business Process Owners today and there was indeed a consensus that it should be possible to differentiate requirements based on mobility type. However, it doesn't seem to be a burning need and could be kept for a next version of the API.

**Comment by jiripetrzelka** [comment]
Although the specification of requirements for trainees did not seem a "burning need" a year ago, I would like to suggest to return to this issue and upgrade the Factsheet API now before testing of nominations starts so that we don't have to repeat the testing once again when requirements for trainees are introduced.

**Comment by jiripetrzelka** [comment]
Can I assume that all entries published in Factsheet's additional requirements currently apply only to long-term student mobility for study? So, if the nomination is for short-term or traineeship mobility, no additional requirements should be included in the response of omobilities get?

**Comment by jiripetrzelka** [comment]
An example from PROD, DK VEJLE06:

Language certificate - Please see language proficiency requirements in our course catalogue
Transcript of Records - The document must be issued in English
Motivation letter - This is mandatory for traineeship applicants, optional for other applicants
Other document - Criminal record, if the traineeship placement requires it. see course catalogue

I thought that we were aiming at automation, which would enable mobility software to ask students for specific documents and determine at which point all nomination documents are ready to make the nomination.

But now it rather seems that we are headed towards a state in which IROs will be required to manually process other HEIs Factsheets to decide what to ask from students and distinguish not only whether it is a study or traineeship but also whether the document is required or optional.

I suppose as time goes by, there will be much more HEIs with "Other documents" stating specific rules for short-term mobilities and once the need to somehow reflect this in a structured way in the XML specification becomes "burning" according to BPOs, there will already be hundreds of Factsheets in PROD with textual descriptions. Not a very appealing prospect.
